examples/s3/bucketpolicy.yaml

```diff
@@ -21,7 +20,7 @@ spec:
         principal:
           allowAnon: true # this boolean flag makes the bucket public, this takes priority over the aws principal.
         resource:
-          - test.s3bucket.aws.crossplane.io
+          - test-bucket-mvf
```

Comment:
Just wanted to confirm this is a valid value here?

Reply:
Nope, thanks for catching!

Reply:
@muvaf looks like this may not have been updated yet?

Reply:
@hasheddan fixed now